src/js/helpers/Util.js

```diff
@@ -1,6 +1,6 @@
 var consecutiveNumber = 0;
 var objectPath = require("object-path");
-
+var toString = Object.prototype.toString;
```

Comment:
The ``Util`` was using the *global* (``window.toString``) method, which is bound to the ``window`` object and can't be *call*ed on a different objects in IE. The respective line is just used to init/set ``toString`` to the the base ``Object.prototype.toString`` method... I'll add a comment.